sphynx/lynxkite-sphynx/main.go

```diff
@@ -220,12 +219,10 @@ func main() {
 			return http.ListenAndServe(fmt.Sprintf(":%s", debugPort), nil)
 		}()
 	}
-	keydir := flag.String(
-		"keydir", "", "directory of cert.pem and private-key.pem files (for encryption)")
-	flag.Parse()
+	keydir := os.Getenv("SPHYNX_CERT_DIR")
```

Comment:
Everything else was passed through environment variables but this one was a flag for some reason. Environment variables are nicer in that they are passed automatically to child processes.